mimecast/komand_mimecast/connection/connection.py

```diff
@@ -6,6 +6,9 @@
 
 
 class Connection(komand.Connection):
+    _get_managed_url_URI = "/api/ttp/url/get-all-managed-urls"
+    _delete_managed_url_URI = '/api/ttp/url/delete-managed-url'
+    _create_managed_url_URI = '/api/ttp/url/create-managed-url'
```

Comment:
All three variables should be public constants (right now they're private) - example: `GET_MANAGED_URL_URI`

Reply:
I agree Matt, however I made this change because in order to implement the DELETE action properly, a GET_MANAGED_URL action must be taken.